### ISSUE TITLE
Add a sample for TypeScript applications

### ISSUE THREAD

**Issue by subesokun** [dapr/samples#157]
## Describe the proposal

It'd be great if there is also a sample available for a TypeScript based Dapr application.

Currently I'm running the app via

```json
{
  "name": "my-app",
  ...
  "scripts": {
    "dapr": "dapr run --app-id my-app --app-port 3000 ts-node-dev lib/main.ts"
  },
  "dependencies": {
    ...
  },
  "devDependencies": {
    "ts-node-dev": "^1.0.0-pre.44",
    ...
  }
}
```

```bash
npm run dapr
```

but not sure if this is the most elegant / recommended way for running TS based Dapr applications locally.

Btw., having the samples repo and the `docs/howto` section (which again contains samples) is very confusing as I've to search always in two places.

**Comment by RicardoNiepel** [comment]
SDK specific examples can be found in the SDK repos - this also applies to typescript:
https://github.com/dapr/js-sdk/tree/master/examples/typescript

**Open items:**
* adding readme to the existing typescript sample
* adding link to the typescript sample from the JS-SDK root
* improving the existing TS sample with hot reload / real-live TS environment

I think we don't need a full TS sample with all building blocks - it should just be a Getting Started with all TS best practices in it using just just one Dapr building block and enable a good development/debugging experience with the Dapr sidecar.
@subesokun Your thoughts on it?

**Comment by subesokun** [comment]
@RicardoNiepel Exactly, the example should just be a simple "Hello World" Dapr application that covers your listed open items.

**Comment by msfussell** [comment]
It would be good to have an example in the docs in the howto section on how to use Dapr as a Typescript developer. @subesokun Would you be able to provide a doc showing how to use Typescript that we can include in the docs repo?

**Comment by subesokun** [comment]
@msfussell Sure, I'll try to prepare a sample TypeScript project with docu and share it here.

**Comment by subesokun** [comment]
@msfussell I've created now a small TypeScript howto/demo as a private contribution from my side :) I was also experimenting with the project setup to improve the development experience but many aspects are still missing I guess. But as a starting point that might be sufficient. I've tried also an alternative approach for the VS Code debugging setup so that I can avoid setting a fixed port or app ID in the VSCode config.

Let me know what you think :) https://github.com/dapr/docs/pull/602

**Comment by youngbupark** [comment]
@wcs1only Can you please review @subesokun 's PR ?
